File: Backend/AICareerHub.API/AICareerHub.API/Models/ResumeProject.cs
namespace AICareerHub.API.Models
{
    public class ResumeProject
    {
        public Guid Id { get; set; }

        public Guid ResumeId { get; set; }

        public string Name { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public string Technologies { get; set; } = string.Empty;

        public string? ProjectUrl { get; set; }

        public Resume Resume { get; set; } = null!;
    }
}

File: Frontend/aicareerhub-ui/src/app/core/models/resume-project.ts
export interface ResumeProject {
  id: string;
  name: string;
  description: string;
  technologies: string;
  projectUrl?: string | null;
}

export interface ResumeProjectRequest {
  name: string;
  description: string;
  technologies: string;
  projectUrl?: string | null;
}